edb/api/errors.txt

```diff
@@ -24,6 +24,7 @@
 ####
 
 0x_02_00_00_00   UnsupportedFeatureError
+0x_02_00_01_00   UnsupportedBackendFeatureError
```

Comment:
I'd consider making it a top-level separate type.

Reply:
Changed.  I added a new base `BackendError` and inherited `UnsupportedBackendFeatureError` from it.